staking_deposit/cli/generate_bls_to_execution_change.py

```diff
@@ -0,0 +1,195 @@
+)
+from staking_deposit.exceptions import ValidationError
+from staking_deposit.utils.intl import (
+    closest_match,
+    load_text,
+)
+from staking_deposit.settings import (
+    ALL_CHAINS,
+    MAINNET,
+    PRATER,
+    get_chain_setting,
+    get_devnet_chain_setting,
+)
+from .existing_mnemonic import (
+    load_mnemonic_arguments_decorator,
+)
+
+
+def get_password(text: str) -> str:
+    return click.prompt(text, hide_input=True, show_default=False, type=str)
+
+
+FUNC_NAME = 'generate_bls_to_execution_change'
+
+
+@click.command()
+@jit_option(
+    default=os.getcwd(),
+    help=lambda: load_text(['arg_bls_to_execution_changes_folder', 'help'], func=FUNC_NAME),
+    param_decls='--bls_to_execution_changes_folder',
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda x: closest_match(x, list(ALL_CHAINS.keys())),
+        choice_prompt_func(
+            lambda: load_text(['arg_chain', 'prompt'], func=FUNC_NAME),
+            list(ALL_CHAINS.keys())
+        ),
+    ),
+    default=MAINNET,
+    help=lambda: load_text(['arg_chain', 'help'], func=FUNC_NAME),
+    param_decls='--chain',
+    prompt=choice_prompt_func(
+        lambda: load_text(['arg_chain', 'prompt'], func=FUNC_NAME),
+        # Since `prater` is alias of `goerli`, do not show `prater` in the prompt message.
+        list(key for key in ALL_CHAINS.keys() if key != PRATER)
+    ),
+)
+@load_mnemonic_arguments_decorator
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda num: validate_int_range(num, 0, 2**32),
+        lambda: load_text(['arg_validator_start_index', 'prompt'], func=FUNC_NAME),
+    ),
+    default=0,
+    help=lambda: load_text(['arg_validator_start_index', 'help'], func=FUNC_NAME),
+    param_decls="--validator_start_index",
+    prompt=lambda: load_text(['arg_validator_start_index', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda validator_indices: validate_validator_indices(validator_indices),
+        lambda: load_text(['arg_validator_indices', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_validator_indices', 'help'], func=FUNC_NAME),
+    param_decls='--validator_indices',
+    prompt=lambda: load_text(['arg_validator_indices', 'prompt'], func=FUNC_NAME),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda bls_withdrawal_credentials_list:
+            validate_bls_withdrawal_credentials_list(bls_withdrawal_credentials_list),
+        lambda: load_text(['arg_bls_withdrawal_credentials_list', 'prompt'], func=FUNC_NAME),
+    ),
+    help=lambda: load_text(['arg_bls_withdrawal_credentials_list', 'help'], func=FUNC_NAME),
+    param_decls='--bls_withdrawal_credentials_list',
+    prompt=lambda: load_text(['arg_bls_withdrawal_credentials_list', 'prompt'], func=FUNC_NAME),
+)
+# TODO: retype confirmation
+@jit_option(
+    callback=captive_prompt_callback(
```

Comment:
(As discussed in person), This should be a captive prompt that requires verification from the user. Much like how we handle mnemonics and be able to be skipable with `--non_interactive`